src/Stratis.Bitcoin.Features.PoA/FederationManager.cs

```diff
@@ -88,13 +88,17 @@ public void Initialize()
             {
                 string message = "Key provided is not registered on the network!";
 
-                this.logger.LogCritical(message);
-                throw new Exception(message);
+                this.logger.LogWarning(message);
```

Comment:
So you agree with my previous suggestion to start a node even when the key is not yet registered?

Reply:
Yes. 
After this PR you can start the node and sync up until the point key is registered and after that mining will start automatically. 

Also if fed member is kicked it will not lead to fed member's node throwing.